src/pages/basic_data/app_admin/central_import/CentralImportPage.tsx

```diff
@@ -84,15 +73,15 @@ export const CentralImportPage = ({ statusFilter, yearFilter }: CentralImportPag
       ) : searchResults.length > 0 ? (
         <ListPagination
           count={importCandidateQuery.data?.totalHits ?? 0}
-          rowsPerPage={rowsPerPage}
+          rowsPerPage={sizeParam}
           page={page + 1}
-          onPageChange={(newPage) => updatePath(((newPage - 1) * rowsPerPage).toString(), rowsPerPage.toString())}
+          onPageChange={(newPage) => updatePath(((newPage - 1) * sizeParam).toString(), sizeParam.toString())}
           onRowsPerPageChange={(newRowsPerPage) => updatePath('0', newRowsPerPage.toString())}
           showPaginationTop
           sortingComponent={
             <SortSelector
-              sortKey={SearchParam.SortOrder}
-              orderKey={SearchParam.OrderBy}
+              sortKey={ImportCandidatesSearchParam.SortOrder}
+              orderKey={ImportCandidatesSearchParam.OrderBy}
               aria-label={t('search.sort_by')}
               size="small"
               variant="standard"
```

Comment:
Ikke lenger støtte for `importStatus.modifiedDate` som brukes under her. Venter avklaring fra @StigNorland for hva vi gjør her

Reply:
tar vekk støte for `importStatus.modifiedDate` foreløpig